indy/Dockerfile

```diff
@@ -7,23 +7,36 @@ EXPOSE 8080 8081 8000
 USER root
 
 ADD start-indy.py /usr/local/bin/start-indy.py
+ADD setup-user.sh /usr/local/bin/setup-user.sh
+ADD setup-user.sh /etc/profile.d/setup-user.sh
```

Comment:
Why does it need to be in both locations? I realize these won't actually fall out of sync, since this is an image build, but it does look a little strange.

Reply:
Probably unnecessary duplication.